src/main/java/com/puppycrawl/tools/checkstyle/site/XdocsTemplateSink.java

```diff
@@ -32,14 +33,22 @@
  * A sink for Checkstyle's xdoc templates.
  * This module will be removed once
  * <a href="https://github.com/checkstyle/checkstyle/issues/13426">#13426</a> is resolved.
+ * This class is required until
+ * <a href="https://issues.apache.org/jira/browse/DOXIA-707">DOXIA-707</a> is resolved.
  *
  * @see <a href="https://maven.apache.org/doxia/doxia/doxia-sink-api">Doxia Sink API</a>
  */
 public class XdocsTemplateSink extends XdocSink {
 
+    /** Linux Style End of Line. */
+    private static final String LINUX_EOL = "\n";
```

Comment:
This is Unix EOL, not Linux.

Reply:
I thought it is same, what is Linux eol?

Reply:
Better to just call it what it is,`NEWLINE`.

Reply:
https://en.m.wikipedia.org/wiki/Newline , looks like eol in well used also, newline will require "character" word to make it clear what it is.
`NEWLINE_CHARACTER`.

Jdk use different wording for this `System.getProperty(“line.separator”).`

There is no consistency and I am not sure what is better term.

Here we operate in doxia terminology base.
```
@Override
    protected void writeEOL() {
```

Let's keep eol.

Reply:
`NEWLINE_CHARACTER` is fine, terms like `EOL` and `line.separator` are generic on purpose since these could refer to any line break; here we have no need for such ambiguity. We know we are dealing with a newline character, let's be clear.

Reply:
Ok.